config.c

```diff
@@ -326,6 +326,8 @@ static int pv_config_load_file(char *path, struct pantavisor_config *config)
 		&config_list, "system.mediadir", "/media");
 	config->sys.confdir = config_get_value_string(
 		&config_list, "system.confdir", "/configs");
+	config->sys.mdev =
+		config_get_value_bool(&config_list, "system.mdev", false);
```

Comment:
sorry for being so annoying with the names. Can we rename this to system.driver.mdev or system.drivers.mdev?

Reply:
yes lets rename